libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -975,6 +996,43 @@ void AP_GPS_UBLOX::log_rxm_rawx(const struct ubx_rxm_rawx &raw)
 }
 #endif // UBLOX_RXM_RAW_LOGGING
 
+#if UBLOX_RXM_RTCM_LOGGING
+void AP_GPS_UBLOX::log_rxm_rtcm(const struct ubx_rxm_rtcm &rtcm)
+{
+#if HAL_LOGGING_ENABLED
```

Comment:
`UBLOX_RXM_RTCM_LOGGING` should be contingent on this, so this line shouldn't be required

Reply:
Nope, you are wrong